examples/e2e-tests/kubernetes/zones/definition.json

```diff
@@ -1,10 +1,6 @@
 {
     "apiVersion": "vlabs",
     "properties": {
-        "orchestratorProfile": {
-            "orchestratorType": "Kubernetes",
-            "orchestratorRelease": "1.16"

```

Comment:
With some of these removals, you left `"orchestratorRelease": "1.16"` intact, but with others we've removed the whole `"orchestratorProfile"` stanza including the release. I'm assuming the default release is fine for these tests, but I just wanted to point it out and make sure it was intentional.

Reply:
That's right, I tried to use common sense and remove the release property when it was *only* complemented by the type property. I think there's follow-up work to remove the remaining version properties, unless they are actually meaningful (like, this is the minimum version you need for a particular config).